docs/ext/django/django.rst

```diff
@@ -0,0 +1,7 @@
+OpenTelemetry Django Instrumentation
```

Comment:
Probably this file should not be in this PR.

Reply:
Removed